libs/platform/user/kernel_um.cpp

```diff
@@ -563,6 +583,10 @@ RtlSetDaclSecurityDescriptor(
     _In_opt_ PACL Dacl,
     _In_ BOOLEAN DaclDefaulted)
 {
+    if (ebpf_fault_injection_inject_fault()) {
+        return STATUS_INVALID_PARAMETER;
```

Comment:
See earlier comment about INVALID_PARAMETER

Reply:
Thanks. In this case, I can return 'STATUS_INVALID_SECURITY_DESCR' instead. Or, do you want to remove fault injection from this API?

Reply:
STATUS_INVALID_SECURITY_DESCR means "does not point to a valid security descriptor" which would be a caller bug as I udnerstand it, so that would not be appropriate.   Again, I don't know whether fault injection should be removed or not, I want you (or someone) to tell me what the case to handle is that isn't a caller bug.  If you don't know of one, then I think fault injection is not appropriate.

Reply:
Removed fault injection from this API.